docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -1,16 +1,16 @@
 Contributing example notebooks
 ------------------------------
 
-To contribute new example notebooks please submit a pull request. We have
-certain requirements that need to be satisfied to contribute a notebook. Those
-include:
+We'd love to collaborate with anyone interested in describing scenarios for using Fairlearn!
 
-* Clear structure
-* Appropriate framing of the problem while respecting that fairness is a
-  fundamentally sociotechnical challenge
-* Usage of at least some part of Fairlearn
+When discussing and critiquing examples, here are some lenses to consider:
 
-Good examples of existing notebooks that abide by these requirements are:
+Microrubric for critiques:
```

Comment:
@MiroDudik If you want to quibble on the naming here, go for it! 😄  Or please feel free to rename this to whatever you think is a better title :)

Reply:
I'm not sure why there's even a section title here - especially immediately after a colon.